system/classes/Kohana/UTF8.php

```diff
@@ -138,11 +147,13 @@ public static function strip_non_ascii($str)
 	 *     $ascii = UTF8::transliterate_to_ascii($utf8);
 	 *
 	 * @author  Andreas Gohr <andi@splitbrain.org>
-	 * @param   string  $str    string to transliterate
-	 * @param   integer $case   -1 lowercase only, +1 uppercase only, 0 both cases
+	 * @param   string  $str           string to transliterate
+	 * @param   integer $case          -1 lowercase only, +1 uppercase only, 0 both cases
+	 * @param   array   $lower_accents lower accents
+	 * @param   array   $upper_accents upper accents
 	 * @return  string
 	 */
-	public static function transliterate_to_ascii($str, $case = 0)
+	public static function transliterate_to_ascii($str, $case = 0, array $lower_accents = NULL, array $upper_accents = NULL): string
```

Comment:
Why do you explicitly specify the return type? As far as I understand, in Koseven this can be done only in special cases.

Reply:
we cant update typing for all classes, can do it step by step.

Reply:
To implement such changes, you need to add a new feature branch and work there, and then merge the branches.